sdcm/stress/latte_thread.py

```diff
@@ -175,13 +188,16 @@ def _run_stress(self, loader, loader_idx, cpu_idx):
 
         if not os.path.exists(loader.logdir):
             os.makedirs(loader.logdir, exist_ok=True)
+
+        stress_operation = get_latte_operation_type(self.stress_cmd)
+        first_tag_or_op = "-" + (find_latte_tags(self.stress_cmd) or [stress_operation])[0]
         log_file_name = os.path.join(
-            loader.logdir, 'latte-l%s-c%s-%s.log' % (loader_idx, cpu_idx, uuid.uuid4()))
-        LOGGER.debug('latte benchmarking tool local log: %s', log_file_name)
+            loader.logdir, 'latte%s-l%s-c%s-%s.log' % (first_tag_or_op, loader_idx, cpu_idx, uuid.uuid4()))
+        # TODO: return back to be "debug" after finishing debugging
+        LOGGER.info('latte benchmarking tool local log: %s', log_file_name)
```

Comment:
finished?

Reply:
yes, seems like it should be put back to debug

Reply:
Done